test/integration/kubernetes_deploy_test.rb

```diff
@@ -195,4 +195,13 @@ def test_wait_false_ignores_non_priority_resource_failures
     assert_equal 1, pods.size, "Unable to find web pod"
     assert_equal "Pending", pods.first.status.phase
   end
+
+  def test_extra_bindings_should_be_rendered
```

Comment:
Can we add a test case for a binding referenced by YAML but not passed from CLI? How would the error look like?